### ISSUE TITLE
No highlighting and text colour in dark/spacegray theme

### ISSUE THREAD

**Issue by mevers** [HappenApps/Quiver#1394]
Hi @ylian 

Highlighting and manually changing the text colour has no effect when using the dark or spacegray themes. This has been around for a fairly long time, and is a fairly serious "bug" as it severely reduces the usability of the dark and spacegray modes. There were at least two open (!) issues here that referred to the same problem (I had commented on one of them). **For some reason they were deleted without any further message/clarification.** Can you clarify *why* these issues were quietly deleted even though the issue is still there?

To reproduce, goto Quiver → Preferences → Themes → Dark, then highlight any text in the main body of the note. Highlighting is not visible

Without highlighting "Some sample text"
![Screen Shot 2019-11-14 at 8 18 58 am](https://user-images.githubusercontent.com/9429788/68805279-9530b300-06b7-11ea-8841-93e22448f10f.png)

After highlighting "Some sample text"
![Screen Shot 2019-11-14 at 8 18 29 am](https://user-images.githubusercontent.com/9429788/68805298-9f52b180-06b7-11ea-9ea9-7eac08d4c39a.png)

**Comment by ylian** [comment]
This issue is much harder to fix than it looks. Highlighting uses a default color which is not customizable, so I had to disable highlighting in dark themes.

**Comment by mevers** [comment]
It's not only the highlighting; setting the text colour manually also does not work. Is there any workaround, or will this be fixed in the upcoming end-of-November version?

Considering your response, why delete the previous issues without any further comments? I think this does not help with creating an open and transparent environment to provide updates for a piece of software that you charge for.

I like Quiver and support your work. Deleting posts/issues is not the right way.

**Comment by ylian** [comment]
This is a misunderstanding. I didn't delete any issue. Either they are getting hard to find or maybe the original author of the issue deleted it.

**Comment by ylian** [comment]
The reason is the same why setting text color doesn't work. I had to force the text color in dark themes otherwise text could become unreadable when a user switches from a bright theme to a dark theme.

**Comment by mevers** [comment]
Ok, in that case this is my bad and I apologise for accusing you of deleting posts.

Since you mention this being harder to fix than it looks, will this be addressed in the upcoming version? It seems that more and more users are switching to dark themes, so it would be very nice to have a working dark model in Quiver.

**Comment by ylian** [comment]
I have a couple ideas that I will try. But no, this is not part of the major rewrite I have been working on. I will certainly look into this again once the big rewrite is done.

**Comment by mevers** [comment]
Ok. I appreciate the update.

**Comment by zombillano** [comment]
Any update on this? I discovered this app today and instantly fell in love with it, but I found so frustrating not being able to highlight using the dark theme :(

**Comment by ijoseph** [comment]
It's possible using custom CSS in the Markdown mode at least… I can share how if you're interested.
